docs/courses/quality_issues_with_morphometric.rst

```diff
@@ -0,0 +1,43 @@
+.. _quality_issues_with_morphometric_data:
```

Comment:
I think as long as it is updated with my past branch, everything should flow together, as the toctree is done correctly but I am running into issues because the branches I am pushing to are different.